### DISCUSSION TITLE
Image Component support alternative (mobile) image source

### DISCUSSION THREAD

**Discussion by dohomi** [vercel/next.js#18358] [Ideas]
# Feature request

## Is your feature request related to a problem? Please describe.

I would love to see an alternative image source tag on the <Image /> component. This source should be used on mobile (portrait) devices instead of the default source

## Describe the solution you'd like
```js
<Image src="/path/to/image-3x2.png" srcMobile="/path/to/image-2x3.png" />
```

## Additional context

This would enable lets say a 3:2 image ratio for tablet/desctop and 2:3 image ratio for mobile devices

**Comment by ettaibi** [comment]
+1
